internal/charmstore/migrations.go

```diff
@@ -138,6 +142,36 @@ func addSupportedSeries(db StoreDatabase) error {
 	return nil
 }
 
+// handleDevelopmentEntities adds the Development field to all entities and
```

Comment:
handle doesn't seem to be the correct verb here perhaps it should be two migrations, addDevelopment & addDevelopmentACLs